docker/README.md

```diff
@@ -124,8 +124,9 @@ Posible options for the oneliner are:
 * -d or --desktopicon       Generates a desktop icon to launch the hand (default: true)
 * -b or --configbranch      Specify the branch for the specific hand (Only for dexterous hand)
 * -sn or --shortcutname     Specify the name for the desktop icon (default: Shadow_Hand_Launcher)
-* -o or --optoforce           Specify if optoforce sensors are going to be used (default: false)
-* -l or --launchhand          Specify if hand driver should start when double clicking desktop icon (default: true)
+* -o or --optoforce         Specify if optoforce sensors are going to be used (default: false)
+* -l or --launchhand        Specify if hand driver should start when double clicking desktop icon (default: true)
+* -bt or --buildtoolsbranch Specify the Git branch for sr-build-tools (remember to replace # with %23) (default: master)
```

Comment:
aha, we sure will. Please do it automatically :) (I mean replace # to %23 ;) )

Reply:
this is an example how we are doing this in Docker files https://github.com/shadow-robot/sr-build-tools/blob/master/docker/ros/kinetic/base/Dockerfile#L45